chaseadamsio/chaseadams.io
# Add storybook

- add Storybook to project
- add path for TS to Webpack config
- add MDX support to Storybook

# Testing
- on branch, run `yarn storybook`
- see it open a new browser tab
- see the date component

 ✅ dev tested
